lib/buffy/throttle.ex

```diff
@@ -219,6 +223,17 @@ defmodule Buffy.Throttle do
       @impl GenServer
       @spec handle_info(:timeout, Buffy.Throttle.state()) :: {:stop, :normal, Buffy.Throttle.state()}
       def handle_info(:timeout, {key, args}) do
+        selected_jitter = 1 - :rand.uniform(unquote(jitter) + 1)
+
+        :telemetry.execute([:buffy, :throttle, :handle, :jitter], %{
+          args: args,
+          jitter: selected_jitter,
+          key: key,
+          module: __MODULE__
+        })
+
+        Process.sleep(selected_jitter)
```

Comment:
Should we only call this when it's >0? 🤔

Reply:
I'm assuming it's a no-op if it's 0. Should probably add a test there...

Reply:
Added test. Confirmed that `Process.sleep(0)` is a no-op